glyphs-reader/src/font.rs

```diff
@@ -77,20 +77,42 @@ pub enum Shape {
 // The font you get directly from a plist, minimally modified
 // Types chosen specifically to accomodate plist translation.
 #[derive(Debug, FromPlist, PartialEq, Eq)]
+#[allow(non_snake_case)]
 struct RawFont {
     pub units_per_em: Option<i64>,
     pub family_name: String,
+    pub copyright: Option<String>,
+    pub designer: Option<String>,
+    pub designerURL: Option<String>,
```

Comment:
do you actually need to `#[allow(non_snake_case)]`?
Can you not do this at the point where you use the attributes:

```diff
- v2_to_v3_name(properties, &self.designerURL, "designerURL");
+ v2_to_v3_name(properties, &self.designer_url, "designerURL");
```

Reply:
No, because the plist parser will fail to populate the field at all.